CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.3.0 (Unreleased)
 
 * Added edge gateway create/delete functions [#130](https://github.com/vmware/go-vcloud-director/issues/130).
+* Added global load balancer configuration support [#219](https://github.com/vmware/go-vcloud-director/pull/219)
```

Comment:
"configuration" is rather generic if you don't know what this issue is about, what about adding more detail like:
`Added global load balancer configuration support (e.g. enable/disable)`

Reply:
Changed. I also added "edge gateway" in that sentence so that it is clearer.